src/screens/CreatePost/index.js

```diff
@@ -278,7 +278,7 @@ const CreatePost = () => {
 
       const responseUpload = await ImageUtils.uploadImage(pathImg);
       setMediaStorage((val) => [...val, newArr]);
-      setDataImage((val) => [...val, responseUpload.data.url]);
+      if (responseUpload?.data?.url) setDataImage((val) => [...val, responseUpload.data.url]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change made here is a good practice. It checks if `responseUpload.data.url` exists before trying to use it, which can prevent potential runtime errors if `responseUpload` does not have the expected structure. This is an improvement in error handling.
```diff
- setDataImage((val) => [...val, responseUpload.data.url]);
+ if (responseUpload?.data?.url) setDataImage((val) => [...val, responseUpload.data.url]);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->